app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/MapFragment.java

```diff
@@ -46,15 +48,16 @@
     private EditText mSearchText;
     private LatLng latLng;
     private View tooltip;
+    private ClusterManager<MyItem> clusterManager;
+    private boolean bool = false;
```

Comment:
When you open the app for the first time it craches because the mapstate manager tries to find a map position while there is non so i used that boolean to skip that part for the first time

Reply:
The bool is used to the app wont crash

Reply:
It should be named differently than `bool`

Reply:
Already ill fix it